AppSettingsManagerAPI/AppSettingsManagerApi.Infrastructure/Migrations/20230227202735_Permissions.Designer.cs

```diff
@@ -0,0 +1,174 @@
+﻿// <auto-generated />
+using System;
+using AppSettingsManagerApi.Infrastructure.MySql;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+
+#nullable disable
+
+namespace AppSettingsManagerApi.Infrastructure.Migrations
+{
+    [DbContext(typeof(SettingsContext))]
+    [Migration("20230227202735_Permissions")]
+    partial class Permissions
+    {
+        protected override void BuildTargetModel(ModelBuilder modelBuilder)
+        {
+#pragma warning disable 612, 618
+            modelBuilder
+                .HasAnnotation("ProductVersion", "6.0.10")
+                .HasAnnotation("Relational:MaxIdentifierLength", 64);
+
+            modelBuilder.Entity("AppSettingsManagerApi.Infrastructure.MySql.Permission", b =>
+                {
+                    b.Property<string>("UserId")
+                        .HasColumnType("varchar(36)");
+
+                    b.Property<string>("SettingGroupId")
+                        .HasColumnType("varchar(255)");
+
+                    b.Property<string>("ApprovedBy")
+                        .IsRequired()
+                        .HasColumnType("longtext");
+
+                    b.Property<int>("CurrentPermissionLevel")
+                        .HasColumnType("int");
+
+                    b.Property<int>("RequestedPermissionLevel")
+                        .HasColumnType("int");
+
+                    b.Property<bool>("WaitingForApproval")
+                        .HasColumnType("tinyint(1)");
+
+                    b.HasKey("UserId", "SettingGroupId");
+
+                    b.HasIndex("SettingGroupId");
+
+                    b.ToTable("Permissions");
+                });
+
+            modelBuilder.Entity("AppSettingsManagerApi.Infrastructure.MySql.Setting", b =>
+                {
+                    b.Property<string>("SettingGroupId")
+                        .HasMaxLength(36)
+                        .HasColumnType("varchar(36)");
+
+                    b.Property<int>("Version")
+                        .HasColumnType("int");
+
+                    b.Property<DateTimeOffset>("CreatedAt")
+                        .HasColumnType("datetime(6)");
+
+                    b.Property<string>("CreatedBy")
+                        .IsRequired()
+                        .HasMaxLength(36)
+                        .HasColumnType("varchar(36)");
+
+                    b.Property<string>("Input")
```

Comment:
Could you explain what Input does? The rest are self-explanatory.

Reply:
Sorry, same comment as before